js/ui/map.jsx

```diff
@@ -31,10 +31,16 @@ const RadarMap = ({ mapOptions, children }) => {
 
   useEffect(() => {
     Radar.getLocation().then((result) => {
+      if (!result.location || !result.location.latitude || !result.location.longitude) {
```

Comment:
Is it possible for result to be null?
```js
if (!result || !result.location || !result.location.latitude || !result.location.longitude) {
```

Reply:
yes, if the status isn't `RadarStatusSuccess`

Reply:
Might be simpler to just do:
```js
if (result?.location?.latitude && result?.location?.longitude) {
  setUserLocation({
    latitude: result.location.latitude,
    longitude: result.location.longitude,
   });
}
```